compose.yaml

```
@@ -0,0 +1,28 @@
+services:
+  app:
+    container_name: "app"
```

double quotes are not necessary

```suggestion
    container_name: app
```